guardian/production-metrics-lambdas
# Bump to Scala 2.13, SBT 1.8.2 and dependency updates

## What does this change?
This bumps Scala and SBT and upgrades dependencies to the latest stable version wherever possible.

## How to test
This lambda is calls CAPI every 24 hours and posts on a Kinesis stream that is read by editorial-production-metrics, which has a corresponding pr with its own dependency updates: https://github.com/guardian/editorial-production-metrics/pull/247. If you want to see that everything still works as expected, run editorial-production-metrics locally and then run `sbt run` in the root of this project (you'll need composer AWS creds). You should see CAPI being called and the metric events being posted to the Kinesis DEV stream. In the editorial-production-metrics project, you should see these events being processed and then be able to see them in your local application UI.

Alternatively, the branches are both deployed in CODE and you can see them working there.
